### PR TITLE
425 get rid of email js

### LINKED ISSUE #425: Standardize nodemailer vs email js

### PR DESCRIPTION
**Pull request by jpabadir**
close #425

### CODE REVIEW

File: src/back-end/src/email-mentorform.js [hunk 1]
```diff
@@ -28,4 +37,53 @@ module.exports = {
...
+      subject: 'A mentor accepted your mentorship request!',
+      text: `Hi ${
+        parameters.menteeName
+      }, \n\nWe would like to let you know that ${
+        parameters.mentorName
+      } has accepted your mentorship request.\n\nHere is some more information about ${
+        parameters.mentorName
+      }:\n\nHe or she left the following description about themselves for you to read:\n\n${
+        parameters.mentorDescription
+      }\n\nTheir email address is email address is ${
```
Comment (katmartel) [inline src/back-end/src/email-mentorform.js]: Minor typo 
```suggestion
      }\n\nTheir email address is ${
```

File: src/back-end/src/email-mentorform.js [hunk 2]
```diff
@@ -28,4 +37,53 @@ module.exports = {
...
+
+      html: `Hi ${
+        parameters.menteeName
+      }, <br/><br/>We would like to let you know that ${
+        parameters.mentorName
+      } has accepted your mentorship request.<br/><br/>Here is some more information about ${
+        parameters.mentorName
+      }:<br/><br/>He or she left the following description about themselves for you to read:<br/><br/>${
+        parameters.mentorDescription
+      }<br/><br/>Their email address is email address is ${
```
Comment (katmartel) [inline src/back-end/src/email-mentorform.js]: Same as above, there's just a slight typo here

File: src/back-end/src/email-mentorform.js [hunk 3]
```diff
@@ -28,4 +37,53 @@ module.exports = {
...
+        parameters.skills,
+      )}.<br/><br/>Sincerely,<br/><br/>the MentorMatch team`,
+    });
+  },
+
+  welcomeMentee(menteeEmailAddress, menteeName) {
+    transporter.sendMail({
+      from: '"MentorMatch Team" <pacementormatch@gmail.com>',
+      to: menteeEmailAddress,
+      subject: 'A mentor accepted your mentorship request!',
```
Comment (katmartel) [inline src/back-end/src/email-mentorform.js]:
```suggestion
      subject: 'Welcome to MentorMatch!',
```

**Review by katmartel — APPROVED** [review]
Thanks for changing emailjs to nodemailer! I've left some very minor comments for some typos. Great work :)

**Review by aleksssk — APPROVED** [review]
Nice work

**Pull request merged**